apps/researcher/src/app/[locale]/(objects)/objects/[id]/metadata.tsx

```diff
@@ -1,88 +1,186 @@
         {t.rich('noData', {
-          subject: () => <span className="lowercase">{t(identifier)}</span>,
+          subject: () => <span className="lowercase">{t(translationKey)}</span>,
         })}
       </div>
     );
   }
 
+  const creator = attributionId
+    ? await getCommunityByAttributionId(attributionId)
+    : organization;
+
   return (
-    <div className="border-neutral-200 text-neutral-900 rounded border-t flex flex-col lg:flex-row justify-between gap-2 lg:divide-x divide-sand-100">
-      <div className="w-full lg:w-3/5  py-3">{children}</div>
-      <div className="w-full lg:w-2/5 px-2 py-3 text-xs text-sand-700 mt-1 flex flex-row justify-between gap-2">
-        <div className="flex flex-col gap-1">
-          {organization && (
-            <div>
-              {t.rich('addedBy', {
-                name: () => <strong>{organization.name}</strong>,
-                location: () => (
-                  <em>
-                    {organization.address?.addressLocality},{' '}
-                    {organization.address?.addressCountry}
-                  </em>
-                ),
-              })}
-              {/*
-            TODO: Add a 'changed on' date
-            on <em>12-12-2018</em>
-            */}
-            </div>
-          )}
-          {isCurrentPublisher && <div>{t('currentPublisher')}</div>}
+    <div className="border-t border-blueGrey-100 flex flex-col lg:flex-row justify-between gap-2">
+      <div className="w-full lg:w-2/3 py-3 px-2">{children}</div>
+      <div
+        className={classNames(
+          'px-2 py-3 text-xs my-1 self-start w-full lg:w-1/3',
+          {
+            'text-blueGrey-700': isCurrentPublisher,
+            'bg-[#f3eee2] text-sand-800': !isCurrentPublisher,
+          }
+        )}
+      >
+        <div>
+          {t.rich('addedBy', {
+            name: () => <strong>{creator?.name}</strong>,
+          })}
         </div>
+        {(dateCreated || citation) && (
+          <div className="flex justify-between">
+            {dateCreated &&
+              format.dateTime(dateCreated, {
+                dateStyle: 'medium',
+              })}
+            {citation && (
+              <div>
+                <SlideOutButton
+                  id={`${translationKey}-${dateCreated}-citation`}
+                  className="p-1 rounded hover:bg-sand-200 -mt-1"
+                >
+                  <InformationCircleIcon className="w-5 h-5 stroke-sand-700" />
+                </SlideOutButton>
+              </div>
+            )}
+          </div>
+        )}
+        {citation && (
+          <SlideOut id={`${translationKey}-${dateCreated}-citation`}>
+            {citation}
+          </SlideOut>
+        )}
       </div>
     </div>
   );
 }
+
+export async function MetadataEntries({children}: {children: ReactNode}) {
+  const {enrichmentIdentifier} = useMetadata.getState();
+
+  const enrichments: Enrichment[] | undefined = enrichmentIdentifier
```

Comment:
Out of curiosity: is the explicit type (`Enrichment[] | undefined`) in const `enrichments` required (or can TypeScript figure this out itself)?

Reply:
It is not needed, I will remove this type.